oshmem/mca/spml/ucx/spml_ucx.c

```diff
@@ -723,12 +725,18 @@ int mca_spml_ucx_put(shmem_ctx_t ctx, void* dst_addr, size_t size, void* src_add
 #if HAVE_DECL_UCP_PUT_NB
     request = ucp_put_nb(ucx_ctx->ucp_peers[dst].ucp_conn, src_addr, size,
                          (uint64_t)rva, ucx_mkey->rkey, opal_common_ucx_empty_complete_cb);
-    return opal_common_ucx_wait_request(request, ucx_ctx->ucp_worker, "ucp_put_nb");
+    res = opal_common_ucx_wait_request(request, ucx_ctx->ucp_worker, "ucp_put_nb");
 #else
     status = ucp_put(ucx_ctx->ucp_peers[dst].ucp_conn, src_addr, size,
                      (uint64_t)rva, ucx_mkey->rkey);
-    return ucx_status_to_oshmem(status);
+    res = ucx_status_to_oshmem(status);
 #endif
+
+    if ((OSHMEM_SUCCESS == res) && mca_spml_ucx.synchronized_quiet) {
```

Comment:
check mca_spml_ucx.synchronized_quiet first, and use OPAL_UNLIKELY, to reduce branch impact